Essentials/src/com/earth2me/essentials/commands/Commandnear.java

```diff
@@ -92,13 +96,19 @@ private String getLocal(final Server server, final User user, final long radius)
 
                 final long delta = (long) playerLoc.distanceSquared(loc);
                 if (delta < radiusSquared) {
-                    if (output.length() > 0) {
-                        output.append(", ");
-                    }
-                    output.append(player.getDisplayName()).append("§f(§4").append((long) Math.sqrt(delta)).append("m§f)");
+                    nearbyPlayers.offer(player);
                 }
             }
         }
+
+        while (!nearbyPlayers.isEmpty()) {
+            if (output.length() > 0) {
+                output.append(", ");
+            }
+            User nearbyPlayer = nearbyPlayers.poll();
+            output.append(nearbyPlayer.getDisplayName()).append("§f(§4").append((long) Math.sqrt(nearbyPlayer.getLocation().distanceSquared(loc))).append("m§f)");
```

Comment:
Why not just `nearbyPlayer.getLocation().distance(loc)`?

Reply:
I don't think anyone would mind if you changed the code to be more efficient.

Reply:
Feel free to update this PR - we'll re-review it.

Reply:
Ok I improved that now based on Ichbinjoe's suggestion, and quickly checked to make sure it did indeed retain the same behaviour (just in case). Looks fine to me.